uiowa/uiowa
# As a site user, I can add an image block

Addresses #1931 

### How to test:

1. `blt frontend`
2. `blt ds --site=default`
3. `drush @default.local cim`
4. As a webmaster/publisher/editor, add an image block to a page.
5. Try adding/removing a link to make sure the image is linked or unlinked properly.
6. Try out all 3 image styles available, you should see these visually change in aspect ratio but otherwise span full width of the column it is placed in.
7. See how the block interacts with other blocks around it. Does it need padding/margin styles?
